docs/tools.md

```diff
@@ -0,0 +1,378 @@
+# Containerbase Tools
+
+---
+
+- [Prepare](#prepare)
+  - [prepare-tool](#prepare-tool-command)
+- [Install](#install)
+  - [install-tool](#install-tool-command)
+  - [install-gem](#install-gem-command)
+  - [install-npm](#install-npm-command)
+  - [install-pip](#install-pip-command)
+- [Uninstall](#uninstall)
+  - [uninstall-tool](#uninstall-tool-command)
+  - [uninstall-gem](#uninstall-gem-command)
+  - [uninstall-npm](#uninstall-npm-command)
+  - [uninstall-pip](#uninstall-pip-command)
+
+---
+
+## Prepare
+
+### `containerbase-cli prepare tool` <a name="prepare-tool-command"></a>
+
+#### Alias <a name="Alias-prepare-tool"></a>
+
+`prepare-tool`
+
+#### Description <a name="Description-prepare-tool"></a>
```

Comment:
why the anchors? 🤔

Reply:
Each command has `Alias` , `Description`... and that's why added this:
```
"no-duplicate-heading": {
      "siblings_only": true,
    },
```